repos/fdbt-site/src/design/modules/_spinner.scss

```diff
@@ -0,0 +1,86 @@
+
+.lds-spinner {
+    color: white;
+    display: inline-block;
+    position: relative;
+    width: 80px;
+    height: 80px;
+}
+.lds-spinner div {
```

Comment:
Is it worth refactoring this and putting it inside the top level class above? What do you think @JwanKhalaf ?

Reply:
Done!